modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/LongTextObjectFieldBusinessType.java

```diff
@@ -82,4 +93,30 @@ public String getName() {
 		).build();
 	}
 
+	@Override
+	public Set<String> getRequiredSettings() {
+		return SetUtil.fromArray("showCounter");
+	}
+
+	@Override
+	public void validateObjectFieldSettings(
+			String objectFieldName, Map<String, String> objectFieldSettings)
+		throws PortalException {
+
+		ObjectFieldBusinessType.super.validateObjectFieldSettings(
+			objectFieldName, objectFieldSettings);
+
+		String maxLengthString = objectFieldSettings.get("maxLength");
+
+		if (Validator.isNotNull(maxLengthString)) {
```

Comment:
@rodrigopaulino just one question, do we need to validate the `showCounter`?

Reply:
Now that you mentioned it. We should verify if it's a boolean value. Thanks again, @carolmariaabb

Reply:
Resolved and reviewed offline